docs/_posts/2013-10-3-community-roundup-9.md

```diff
@@ -56,7 +56,7 @@ The video will be available soon on the [JSConf EU website](http://2013.jsconf.e
 
 [Todd Kennedy](http://blog.selfassembled.org/) working at [Cond&eacute; Nast](http://www.condenast.com/) implemented a wrapper on-top of [JSHint](http://www.jshint.com/) that first converts JSX files to JS.
 
-> A wrapper around JSHint to allow linting of files containg JSX syntax. Accepts glob patterns. Respects your local .jshintrc file and .gitignore to filter your glob patterns.
+> A wrapper around JSHint to allow linting of files containing JSX syntax. Accepts glob patterns. Respects your local .jshintrc file and .gitignore to filter your glob patterns.
```

Comment:
It was clearly just a typographical error, and the fix clearly doesn't change the meaning of the sentence.  I agree it's not worth investing a ton of effort in fixing up old blog posts, but if the work is already done and the fix is simple, and the correction makes the original author look good, so I figured we might as well take it.

I make typos all the time, and if someone corrected them when quoting me, so long as they were careful not to change the meaning of my sentence, I would appriciate the correction.

In my opinion, the key here is "what would the original author say?".  Fortunately, we can ask him.  cc @toddself